apps/accounts/models.py

```diff
@@ -1,4 +1,49 @@
 from django.db import models
 
+# Base user class
+class User(models.Model):
+    name = models.CharField(max_length=64, blank=False)             # Ivan
+    surname = models.CharField(max_length=64, blank=False)          # Ivanov
+    patronymic = models.CharField(max_length=64, blank=False)       # Ivanovych
+    email = models.EmailField(max_length=64, blank=False)           # ivanov@gmail.com
+    password = models.CharField(max_length=64, blank=False)         # 21kjfs23iuk
+    avatar = models.ImageField(upload_to='static/images/', blank=True, max_length=1000)     # select image
 
-# Create your models here.
+    # Returns object's name
+    def __str__(self):
+        return(self.name)
```

Comment:
method to return full user name or nickname (selected by user) do not use sensitive data to return